keras/utils/test_utils.py

```diff
@@ -36,7 +36,7 @@ def get_test_data(nb_train=1000, nb_test=500, input_shape=(10,),
 
 
 def layer_test(layer_cls, kwargs={}, input_shape=None, input_dtype=None,
-               input_data=None, expected_output=None, expected_output_dtype=None):
+               input_data=None, expected_output=None, expected_output_dtype=None, with_None=True):
```

Comment:
`with_None` is non-standard style, and not descriptive of what it's supposed to do (also undocumented).